packages/polling-controller/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "@metamask/polling-controller",
+  "version": "0.0.0",
+  "description": "Polling Controller is the base for controllers that polling by networkClientId",
+  "keywords": [
+    "MetaMask",
+    "Ethereum"
+  ],
+  "homepage": "https://github.com/MetaMask/core/tree/main/packages/polling-controller#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/core/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/core.git"
+  },
+  "license": "MIT",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build:docs": "typedoc",
+    "changelog:validate": "../../scripts/validate-changelog.sh @metamask/polling-controller",
+    "publish:preview": "yarn npm publish --tag preview",
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "dependencies": {
+    "@metamask/base-controller": "^3.2.1",
+    "@metamask/controller-utils": "^4.3.2",
+    "@metamask/network-controller": "^12.2.0",
+    "@metamask/utils": "^6.2.0",
+    "@types/uuid": "^8.3.0",
+    "uuid": "^8.3.2"
+  },
+  "devDependencies": {
+    "@metamask/auto-changelog": "^3.1.0",
+    "@types/jest": "^27.4.1",
+    "deepmerge": "^4.2.2",
+    "jest": "^27.5.1",
+    "ts-jest": "^27.1.4",
+    "typedoc": "^0.22.15",
+    "typedoc-plugin-missing-exports": "^0.22.6",
```

These were just updated on main; this will resolve a constraints error upon next branch update:
```suggestion
    "typedoc": "^0.23.15",
    "typedoc-plugin-missing-exports": "^0.23.0",
```

---

The network controller versions will need to be bumped as well, and the controller util versions